src/utils/filterSettings.js

```diff
@@ -47,7 +47,13 @@ export const multiSelectSettings = [
       value: label,
     })),
   },
-]
+].map(setting => {
```

Comment:
this is great!

but I'd also like you to retrieve the filterName from `settings` by using `({ filterName })` as the argument, and then return `{...option, filterName}` as the mutation option object